plugins/aws/fix_plugin_aws/collector.py

```diff
@@ -254,6 +255,11 @@ def get_last_run() -> Optional[datetime]:
                     log.warning(f"Unexpected node type {node} in graph")
                     raise Exception("Only AWS resources expected")
 
+            # add access edges
+            log.info(f"[Aws:{self.account.id}] Create access edges.")
+            access_edge_creator = AccessEdgeCreator(global_builder)
+            access_edge_creator.add_access_edges()
```

Comment:
Consider putting this under a feature flag as long as we are not 100% confident.
Plus: we should have a config option so users can enable/disable this feature.